test/token-paymaster/btpm-coverage-specs.ts

```diff
@@ -33,7 +33,7 @@ const MOCK_VALID_UNTIL = "0x00000000deadbeef";
 const MOCK_VALID_AFTER = "0x0000000000001234";
 const MOCK_SIG = "0x1234";
 const MOCK_ERC20_ADDR = "0x" + "01".repeat(20);
-const MOCK_FEE = "0";
+const DEFAULT_FEE_MARKUP = 1100000;
```

Comment:
I don't see this variable being used in this file.

Reply:
removed, can bring back later